### ISSUE TITLE
Feature Request: Interface Builder Designable

### ISSUE THREAD

**Issue by Fudge0952** [philackm/Scrollable-GraphView#8]
It would be really nice to be able to not only create the object in interface builder but to be able to design the object in interface builder as well.

To fix the first problem you need to replace the `initWithCoder` function with

```Swift
required public init?(coder aDecoder: NSCoder) {`
	super.init(coder: aDecoder)`
}
```

Secondly, after `// MARK: - ScrollableGraphView` you need to mark the class as "designable" by adding `@IBDesignable`. Then for each variable which you wish to expose to interface builder you need to mark the variable with `@IBInspectable`. For example:

```Swift
public var lineWidth: CGFloat = 2
// Will become
@IBInspectable public var lineWidth: CGFloat = 2
```

Finally you need to implement this function:
```Swift
public override func prepareForInterfaceBuilder() {
	//Setup view
}
```

**Comment by philackm** [comment]
Thanks for the help. I've got the basics down and it seems to be working:
<img width="750" alt="screen shot 2016-05-29 at 12 12 00 am" src="https://cloud.githubusercontent.com/assets/2552214/15627782/64316912-2532-11e6-94eb-5a227f5836e3.png">

However there are a whole bunch of bugs regarding the animation and other various layers. I'll need to spend some more time on this when I get a chance.

**Comment by guydaher** [comment]
Any ETA on that? Especially for the part of "creating the object in interface builder", that would be really helpful.

**Comment by philackm** [comment]
No ETA as of yet sorry! I'll hopefully have some time this coming weekend to get this working.

**Comment by andris-zalitis** [comment]
I'm implementing this right now. I was wondering whether you guys have preferences regarding which of the configuration properties should be exposed to IB. There are a lot of them and exposing all may be somewhat cluttering experience.
One approach that I may take - for now expose only those properties which are needed to recreate all the 4 styles of graphs included in the demo.

**Comment by andris-zalitis** [comment]
I ended up adding all of the public properties to IB except for fonts and closures. Even though there are really a lot of them and it's not always easy to find what you need (especially since IB cuts off the longer names), we have used almost all of them in the 4 example graphs. So there was no point of leaving a few out.
#38

**Comment by philackm** [comment]
This has been implemented by @andris-zalitis in pull request #38
